README.md

```diff
@@ -142,7 +142,7 @@ The component accepts the following props:
 |**`pagination`**|boolean|true|Enable/disable pagination
 |**`selectableRows`**|string|'multiple'|Numbers of rows that can be selected. Options are "multiple", "single", "none".
 |**`selectableRowsOnClick`**|boolean|false|Enable/disable select toggle when row is clicked. When False, only checkbox will trigger this action.
-|**`isRowSelectable`**|function||Enable/disable selection on certain rows with custom function. Returns true if not provided.  `function(dataIndex) => bool`
+|**`isRowSelectable`**|function||Enable/disable selection on certain rows with custom function. Returns true if not provided. `function(dataIndex, selectedRows) => bool`. `selectedRows` contains information about selected rows in following format: `{ data // array of data objects, lookup // map of selected ids }`
```

Comment:
Just to keep the descriptions parallel between other areas of documentation, I think we should use this format for the `selectedRows` argument: "object(lookup: {dataindex: boolean}, data: arrayOfObjects: {index, dataIndex})"